### ISSUE TITLE
Tracking related nugets packaged for a cli/msbuild release

### ISSUE THREAD

**Issue by radical** [dotnet/cli#7993]
I am responsible for packaging msbuild for mono. Say, I am looking at tracking the current 15.5 release.
And I need to figure out how to obtain some non-msbuild bits to package with msbuild.

1. I am using https://github.com/dotnet/cli/blob/release/15.5/build/DependencyVersions.props to get the nuget
    versions for the SDKs (https://github.com/dotnet/cli/blob/release/15.5/build/BundledSdks.props).

2. I get the Microsoft.NET.Build.Extensions nuget using https://github.com/dotnet/cli/blob/release/15.5/build/DependencyVersions.props#L17

3. I get NuGet.Build.Tasks with version https://github.com/dotnet/cli/blob/release/15.5/build/DependencyVersions.props#L19

4. I get runtime.osx-x64.Microsoft.NETCore.DotNetHostResolver with version https://github.com/dotnet/cli/blob/release/15.5/build/DependencyVersions.props#L4

Question:

1. I want to find the corresponding nuget version for https://dotnet.myget.org/feed/dotnet-cli/package/nuget/Microsoft.DotNet.MSBuildSdkResolver .
    But I don't know how to relate the cli 15.5 branch with the sdk one and figure out the nuget version I should be using.

**Comment by livarcocc** [comment]
The resolver is produced out of the CLI. Whatever version a CLI has, there will be a matching resolver package for that CLI.

Also note that the latest CLIs coming from release/15.5 are actually versioned 2.1.1-preview. We have abandoned the 15.5 version scheme a while ago.

The latest CLI available from 15.5 is 2.1.1-preview-007139 and there is a matching package for the resolver here: https://dotnet.myget.org/feed/dotnet-cli/package/nuget/Microsoft.DotNet.MSBuildSdkResolver/2.1.1-preview-007139.

**Comment by radical** [comment]
> The latest CLI available from 15.5 is 2.1.1-preview-007139 ..

But how would I know the latest version being used for the particular VS release? And how would I find the corresponding git commit hash, so that I can get to the `cli/build/DependencyVersions.props` to find the versions of the various SDKs?
@akoeplinger found https://dotnetcli.blob.core.windows.net/dotnet/Sdk/release/15.5/latest.version and this has:

```
8b3c80958b8f194b966fab7e82535fcf0af0bf9f
2.1.1-preview-007139
```

That hash gets me to the `cli` repo and I continue from there. But I am wondering if I can rely on this link (https://dotnetcli.blob.core.windows.net/dotnet/Sdk/release/15.5/latest.version) to always represent what is shipping in VS?

**Comment by livarcocc** [comment]
You can't. Not without looking at what is being inserted into VS directly. However, sometimes, a build will rev in the open and be release but will not be immediately injected into VS. As we may have reasons to rev the SDK outside of the VS release cycle.

Separately, we have not been tagging previews that are shipped in VS, but we do tag releases that ship in VS. The tag would tell you the commit. However, even if we did tag them, the tag would only be created on the release day. I don't know if that would be enough antecedence for you or not.

Adding @leecow here who is our release manager. Maybe the solution is to add you or a particular team to be notified when these releases are happening.
